platform-configuration/src/main/java/org/codice/ddf/configuration/ConfigurationManager.java

```diff
@@ -163,11 +167,9 @@
     /**
      * Constructs the list of DDF system Settings (read-only and configurable
      * settings) to be pushed to registered ConfigurationWatchers.
-     * 
-     * @param services
-     *            the list of watchers of changes to the DDF System Settings
-     * @param configurationAdmin
-     *            the OSGi Configuration Admin service handle
+     *
```

Comment:
I actually have JavaDoc formatting turned off because it disagrees with our (fairly rigid) checkstyle on the copyright header. IntelliJ by default will unwrap lines out to the line length limit, apparently Eclipse has a different default. I personally like this behavior better, so if it's not in our formatter profile then I think it should be added.